src/frontend/components/chai-datetime/.bower.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "chai-datetime",
```

Comment:
You can install `chai-datetime` by npm, so no need to copy it's code in component